tests/AbstractFilesystemSymlinkRepositoryTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the puli/repository package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Puli\Repository\Tests;
+
+use Puli\Repository\Resource\DirectoryResource;
+use Symfony\Component\Filesystem\Filesystem;
+use Webmozart\Glob\Test\TestUtil;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ */
+abstract class AbstractFilesystemSymlinkRepositoryTest extends AbstractFilesystemRepositoryTest
```

Comment:
I would name this `AbstractFilesystemRepositorySymlinkTest` to be consistent with the other names:

* We are creating an `Abstract` class
* for the `FilesystemRepository`
* with `Symlink` enabled